src/status_im/ui/screens/subs.cljs

```diff
@@ -86,3 +86,9 @@
          (fn [db [_ item-id]]
            (let [item-animation (get-in db [:chat-animations item-id])]
              (if (some? item-animation) (:delete-swiped item-animation) nil))))
+
+(reg-sub :get-current-account-network
+         (fn [{:keys [network] :as db} [_]]
+           (let [a (get-in db [:account/account :networks network])]
+             #_(js/alert a)
```

Comment:
We can probably simplify this block?

Reply:
Yeah, it's just some leftover debugging crap. I'll remove it